qunit-dom-codemod.js

```diff
@@ -607,6 +607,42 @@ module.exports = function(file, api, options) {
         customMessage ? [newValueNode, customMessage] : [newValueNode]));
     });
 
+  // assert.equal(node.length, 1) -> assert.dom(node).exists({ count: 1 });
```

Comment:
this seems too generic to me. it would basically convert every single array length assertion to a qunit-dom assertion.

Reply:
Ah yeah good point, seems like we can't really know for sure since the node can be any variable. Any ideas?

Reply:
nope, unless we're seeing specifically `findAll('.selector').length` I don't think there is much we can do here